src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * Sorts all persons in the address book and lists all persons to the user.
+ */
+public class SortCommand extends ListCommand {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a list with index numbers sorted by name.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.